travis/docker/Dockerfile

```diff
@@ -78,22 +87,21 @@ RUN echo y | android update sdk --no-ui --all --filter platform-tools,extra-andr
 RUN echo y | android update sdk --no-ui --all --filter android-21
 RUN echo y | android update sdk --no-ui --all --filter sys-img-armeabi-v7a-android-21
 
-# Upgrade pip
-RUN pip install --upgrade pip
-
 #==============================================================================
 #                          PWNTOOLS TEST REQUIREMENTS
 #==============================================================================
 
-# Install pwntools from 'dev', to get all of the latest dependencies
+# Install pwntools from 'dev3', to get all of the latest dependencies
 # Then uninstall pwntools so we have a clean slate, but still have
 # all of its dependencies installed.
 WORKDIR /root
 RUN git clone https://github.com/Gallopsled/pwntools
 WORKDIR /root/pwntools
-RUN pip install --upgrade --editable .
-RUN pip install --upgrade --requirement docs/requirements.txt
-RUN pip uninstall --yes pwntools
+RUN git checkout dev3
+RUN pip3 install --upgrade --editable .
+RUN pip3 uninstall --yes pwntools
+COPY requirements.txt requirements.txt
+RUN pip3 install --upgrade --requirement requirements.txt
```

Comment:
What are these two lines for? We should already install the docs' requirements.txt when running the doctests

Reply:
See comments in travis/docker/requirements.txt above. I updated this logic accordingly.